tensorflow_lite_support/tools/Build_TensorFlow_Lite_Support_Libraries_with_Bazel_and_Colab.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    %env ANDROID_SDK_API_LEVEL=29
</pre><p>We actually use ANDROID_SDK_API_LEVEL=26 internally. And the build tools version we used is 28.0.0. But probably it doesn't matter very much.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tflite-support/pull/620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I think it doesn't matter. It seems to run also with newer version 29 and 29.0.2.